CONTRIBUTING.md

```diff
@@ -55,10 +55,13 @@
 * Avoid introducing trailing whitespace
 * Limit line lengths to 88 characters when possible
 * Write comments to explain non-obvious operations
-* Use whitespaces to improve code readability (examples: after commas; before and
-  after mathematical operators (`+`/`-`/`*`/`/` except `^`), binary operators
-  (`&&`/`||`/...), and comparisons (`<`/`>`/`==`/...); before and after equality
-  signs `=` unless used for the assignment of a default parameter)
+* Use whitespaces to improve code readability. Examples: after commas; before and
+  after binary operators (`&&`/`||`/...), and comparisons (`<`/`>`/`==`/...); before and
+  after equality signs `=` unless used for the assignment of a default parameter. For
+  mathematical operators (`+`/`-`/`*`/`/` except `^`), consider adding whitespace around
+  the operators with the lowest priority. For additional guidance, refer to
+  [Python PEP-8](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements),
```

Comment:
The wording here is pretty subtle, and I'm not sure most readers will catch it. Also, I *think* that in cases without different operator precedence like `x + y + z`, the spaces are more than just something to "consider". If you want anyone to notice this, a couple very simple examples might help, like `x*2 - 1` and `(a+b) * (a-b)` from PEP-8.